astropy/io/misc/asdf/tags/coordinates/earthlocation.py

```diff
@@ -0,0 +1,29 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from asdf.yamlutil import custom_tree_to_tagged_tree
+
+from astropy.coordinates import EarthLocation
+
+from ...types import AstropyType
+
+
+class EarthLocationType(AstropyType):
+    name = 'coordinates/earthlocation'
+    types = [EarthLocation]
+    version = '1.0.0'
+
+
```

Comment:
only one blank line between methods?

Reply:
Yes.  I believe that is the PEP8 recommendation (with 2 lines between functions).